edit/workload/storage/ephemeralVolume/index.vue

```diff
@@ -0,0 +1,81 @@
+<script>
+import LabeledSelect from '@/components/form/LabeledSelect';
+import LabeledInput from '@/components/form/LabeledInput';
+import Checkbox from '@/components/form/Checkbox';
+import Mount from '@/edit/workload/storage/Mount';
+import VolumeMount from '@/edit/workload/storage/volume-mount.js';
+import { mapGetters } from 'vuex';
+
+const drivers = [
```

Comment:
If you want you can read these from the list of components in the folder instead of having to maintain a list.  See https://github.com/rancher/dashboard/blob/master/store/i18n.js#L6-L14